meilisearch-http/tests/content_type.rs

```diff
@@ -110,3 +110,39 @@ async fn error_json_bad_content_type() {
         }
     }
 }
+
+#[actix_rt::test]
+async fn extract_actual_content_type() {
+    let route = "/indexes/doggo/documents";
+    let documents = "[{}]";
+    let server = Server::new().await;
+    let app = test::init_service(create_app!(
```

Comment:
You can get access to a service from the server itself: https://github.com/meilisearch/MeiliSearch/blob/main/meilisearch-http/tests/common/service.rs,

You are actually creating two instances here